app/models/manageiq/providers/hawkular/middleware_manager/live_metrics_capture.rb

```diff
@@ -50,8 +50,8 @@ def collect_live_metrics(metrics, start_time, end_time, interval)
         end
         metrics_ids_map[metric_id] = metric[:name]
       end
-      starts = (start_time - interval).to_i.in_milliseconds
-      ends = end_time.to_i.in_milliseconds + 1
+      starts = start_time.to_i.in_milliseconds
+      ends = (end_time + interval).to_i.in_milliseconds + 1
```

Comment:
I am not sure how this works - doesn't that just move the interval from the start to the end, so when it was before from 22:00 yesterday to 22:00 today, wouldn't this just move from 22:00 today to 22:00 tomorrow?
My understanding is that "show this day" means current day 0:00 until current day 23:59 ?